apps/web/src/components/FormInstance.tsx

```diff
@@ -90,265 +75,130 @@ const FormInstance = ({
       });
   };
 
-  /**
-   * Update the form instance to be marked as completed
-   */
   const _handleFormApprove = async () => {
     if (formInstance.markedCompleted) return;
     completeFormInstanceMutation.mutateAsync(formInstance.id).catch((e) => {
       throw e;
     });
 
-    router.push('/');
+    onClose();
   };
 
   return (
-    <Box>
-      <Flex
-        ml="50px"
-        as="button"
-        align="center"
-        cursor="pointer"
-        mt={10}
-        data-group
-        onClick={router.back}
-      >
-        <LeftArrowIcon
-          boxSize={3}
-          mr={2}
-          fill="var(--mfa-blue-hex)"
-          _groupHover={{ fill: 'var(--chakra-colors-gray-500)' }}
-        />
-        <Text
-          color="var(--mfa-blue-hex)"
-          _groupHover={{ color: 'var(--chakra-colors-gray-500)' }}
-        >
-          Back
-        </Text>
-      </Flex>
-      <Flex align="center" pl={4} mt={4}>
-        <Heading as="h1" ml="30px" mt={4} mr={5}>
+    <Box p={8} bg="#F8F9FA" borderRadius="16px">
+      <Flex justifyContent="space-between" alignItems="center" mb={6}>
+        <Heading as="h1" fontSize="24px">
           {formInstance.name}
         </Heading>
-        <Button
-          variant="link"
-          onClick={() => {}}
-          color="black"
-          fontWeight="normal"
-          textAlign="left"
-          _hover={{ textDecoration: 'none' }}
-          onMouseEnter={() => setIsHovered(true)}
-          onMouseLeave={() => setIsHovered(false)}
-        >
-          <Flex flexDirection="column">
-            <Flex>
-              <PencilIcon mr={1} mt={6} />
-              <Text
-                color="#000"
-                style={{ fontSize: '18px', textDecoration: 'none' }}
-                mt={5}
-              >
-                Edit
-              </Text>
-            </Flex>
-            {isHovered && (
-              <EditUnderlineIcon
-                stroke="black"
-                width="52px"
-                height="2"
-                fill="none"
-              />
-            )}
-          </Flex>
+        <Button variant="ghost" onClick={onClose}
+        h="16px"
+        w="16px">
+        {<CloseIcon />}
```

Comment:
I think the close icon is a little bigger in the design, you should be able to adjust that with width="9.819px" height="9.819px" here.  I don't think it has a hover state either, so maybe add _hover={{backgoundColor: "transparent"}}.